### PR TITLE
PDF - implement multiline text

### PR DESCRIPTION
**Pull request by novsstation**

### CODE REVIEW

File: testing/tests/DevExpress.exporter/jspdfParts/jspdf_v2.dataGrid.tests.js [hunk 1]
```diff
@@ -197,6 +203,336 @@ QUnit.module('Table', moduleConfig, () => {
         });
     });
 
+    QUnit.test('1 col - 1 text line. fontSize default, lineHeight: default', function(assert) {
```
Comment (IgnatovDan) [inline testing/tests/DevExpress.exporter/jspdfParts/jspdf_v2.dataGrid.tests.js]: I think we need a new fixture.js file for each new feature

**Review by IgnatovDan — CHANGES_REQUESTED** [review]

File: testing/tests/DevExpress.exporter/jspdfParts/jspdf_v2.dataGrid.tests.js [hunk 2]
```diff
@@ -197,6 +203,336 @@ QUnit.module('Table', moduleConfig, () => {
...
+            'text,line1,10,45,{baseline:middle}',
+            'setLineWidth,1',
+            'rect,10,15,100,60'
+        ];
+
+        const onExporting = (e) => {
+            e.rowHeight = 60;
+        };
+
+        doc.setFontSize(20);
```
Comment (IgnatovDan) [inline testing/tests/DevExpress.exporter/jspdfParts/jspdf_v2.dataGrid.tests.js]: I think our code should accept a font settings, set its own font and restore initial font at the end.

**Review by IgnatovDan — CHANGES_REQUESTED** [review]

File: testing/tests/DevExpress.exporter/jspdfParts/jspdf_v2.dataGrid.tests.js [hunk 3]
```diff
@@ -197,6 +203,336 @@ QUnit.module('Table', moduleConfig, () => {
...
+        doc.setLineHeightFactor(1);
```
Comment (IgnatovDan) [inline testing/tests/DevExpress.exporter/jspdfParts/jspdf_v2.dataGrid.tests.js]: And here: it's better to pass it as an explicit parameter.

File: testing/tests/DevExpress.exporter/jspdfParts/jspdf_v2.dataGrid.tests.js [hunk 4]
```diff
@@ -197,6 +203,336 @@ QUnit.module('Table', moduleConfig, () => {
...
+            'setLineWidth,1',
+            'rect,10,15,100,60'
+        ];
+
+        const onExporting = (e) => {
+            e.rowHeight = 60;
+        };
+
+        doc.setFontSize(20);
+        doc.setLineHeightFactor(1.5);
```
Comment (IgnatovDan) [inline testing/tests/DevExpress.exporter/jspdfParts/jspdf_v2.dataGrid.tests.js]: I think that these settings can be different for each cell, clients need a way to specify it for entire grid and for each cell

**Review by IgnatovDan — CHANGES_REQUESTED** [review]

**Review by KuznetsovVN — APPROVED** [review]

**Review by IgnatovDan — APPROVED** [review]

**Pull request merged**